data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life-signs aboard, most of which read as likely intelligent, with many thousands more minor life-signs alongside them.`
+			`	Some of these minor life-signs are flora, like what you saw in the massive dome atop the craft, as well as a wide variety of fauna. The rest though seem to be large orbs found throughout the craft, mostly in large groupings. A crew member identifies the orbs as eggs, undoubtedly Korath eggs.`
+			`	As you continue to scan this massive vessel, your computer's analysis makes it clear that a ship of this scale would require an actual army spending weeks of constant fighting to even have a chance of capturing it, and an equally large workforce to fully pilot it. A thorough and time intensive sweep would be absolutely essential to prevent surviving Korath from engaging in guerrilla campaigns against your crew or even just sabotaging vital components.`
+			`	Destroying this ship would be equivalent to blowing up an entire city and certainly greatly upset the Korath. Depending on the Korath's total population, it may even be considered an act of genocide.`
```

Comment:
I think this reads slightly better? Although it's a bit of a matter of preference.

```suggestion
			`	Destroying this ship would be equivalent to blowing up an entire city and would certainly upset the Korath a great deal. Depending on their total population, it may even be considered an act of genocide.`
```

Reply:
Yeah, I will ask that anyone else who reads this just thumb up if you want this change or thumb down for no, since I think both work fine so I will leave it to majority rule

Reply:
Either works. Specifically, I do like the repetition of Korath as an emphasis of who it is we are contemplating the potential large scale deaths of.